animate.css

```
@@ -42,31 +42,31 @@
   -webkit-animation-iteration-count: calc(var(--animate-repeat) * 3);
   animation-iteration-count: calc(var(--animate-repeat) * 3);
 }
-.animate__animated.animate__delay-1s {
+.animate__animated.animate__delay-1x {
```

Could we support the old and the new classes?
```suggestion
.animate__animated.animate__delay-1x,
.animate__animated.animate__delay-1s {